mz_bokeh_package/components/plot_settings.py

```diff
@@ -351,6 +351,14 @@ def _point_shape(self, value: str):
 
             kwargs["fill_alpha"] = fill_alpha
 
+            # Bokeh-problem: color attributes of the selection_glyph and nonselection_glyph are not synced with
+            # the main glyph.
+            # Solution-workaround: initiate an arbitrary color to the two glyphs. The color will be overwritten
+            # in the _point_color function.
+            # See Jira card MZC-1315.
```

Comment:
Is that the actual problem? As far as I understand it you need to make sure that the selection glyph and the nonselection glyph are instantiated when you initialise the glyph. To ensure that, you just assign some dummy value to it. Please check...

I would then also keep that comment short:
```
# Make sure that the selection_glyph and the nonselection_glyphs are instantiated along with the newly created glyph
# See Jira card MZC-1315
```
(But please check that this is the case...)